rules/set-gmsacredentialspec-value/raw.rego

```diff
@@ -0,0 +1,45 @@
+package armo_builtins
+
+import future.keywords.if
+
+# Rule to check if the GMSA credential spec is set in the security context
+deny[msga] {
+    # Verify the object kind
+    obj := input[_]
+    allowed_kinds := {"Pod", "Deployment", "ReplicaSet", "DaemonSet", "StatefulSet", "Job", "CronJob"}
+    allowed_kinds[obj.kind]
+
```

Comment:
Please add OS check using the label `kubernetes.io/os: "windows"` located in `Node` object.